webrtc.html

```diff
@@ -1364,6 +1364,16 @@
             this value is <code>null</code>.
           </dd>
 
+          <dt>static sequence&lt;RTCIceServer&gt; getDefaultIceServers()</dt>
```

Comment:
Can't this be an attribute? e.g. `static readonly attribute FrozenArray<RTCIceServer> defaultIceServers;`